docs/configuration/overview.rst

```diff
@@ -72,30 +71,28 @@ How the Active Side Works
 
 The active side (:ref:`push <job-push>` and :ref:`pull <job-pull>` job) executes the replication and pruning logic:
 
-* Wakeup because of finished snapshotting (``push`` job) or pull interval ticker (``pull`` job).
-* Connect to the corresponding passive side using a :ref:`transport <transport>` and instantiate an RPC client.
-* Replicate data from the sending to the receiving side (see below).
-* Prune on sender & receiver.
+1. Wakeup after snapshotting (``push`` job) or pull interval ticker (``pull`` job).
+2. Connect to the passive side and instantiate an RPC client.
+3. Replicate data from the sender to the receiver.
+4. Prune on sender & receiver.
 
 .. TIP::
-  The progress of the active side can be watched live using the ``zrepl status`` subcommand.
+  The progress of the active side can be watched live using ``zrepl status``.
 
 .. _overview-passive-side--client-identity:
 
 How the Passive Side Works
 --------------------------
 
-The passive side (:ref:`sink <job-sink>` and :ref:`source <job-source>`) waits for connections from the corresponding active side,
-using the transport listener type specified in the ``serve`` field of the job configuration.
-When a client connects, the transport listener performS listener-specific access control (cert validation, IP ACLs, etc)
-and determines the *client identity*.
-The passive side job then uses this client identity as follows:
-
-* The ``sink`` job maps requests from different client identities to their respective sub-filesystem tree ``root_fs/${client_identity}``.
-* The ``source`` might, in the future, embed the client identity in :ref:`zrepl's ZFS abstraction names <zrepl-zfs-abstractions>` in order to support multi-host replication.
+1. The passive side (:ref:`sink <job-sink>` and :ref:`source <job-source>`) waits for connections from the active side,
+on the :ref:`transport <transport>` specified with ``serve`` in the job configuration.
+2. Passive side transport performs listener-specific authentication (determining *client identity*) and authorization (IP ACLs, jobs configs, and more).
```

Comment:
There's a reason that *client identity* is on a separate bullet point, and not in parentheses. The reason is that it's a zrepl specific concept that we should draw attention to.

Reply:
As I read it, it makes sense. It doesn't really appear out of the ordinary, at least when comparing it to the general web. Feel free to improve.